packages/tdigest/tdigest.2.2.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "Simon Grondin"
+authors: [
+  "Simon Grondin"
+  "Will Welch"
+]
+synopsis: "OCaml implementation of the T-Digest algorithm"
+description: """
+The T-Digest is a data structure and algorithm for constructing an approximate distribution for a collection of real numbers presented as a stream.
+
+The T-Digest can estimate percentiles or quantiles extremely accurately even at the tails, while using a fraction of the space.
+
+Additionally, the T-Digest is concatenable, making it a good fit for distributed systems. The internal state of a T-Digest can be exported as a binary string, and the concatenation of any number of those strings can then be imported to form a new T-Digest.
+"""
+license: "MIT"
+homepage: "https://github.com/SGrondin/tdigest"
+dev-repo: "git://github.com/SGrondin/tdigest"
+doc: "https://github.com/SGrondin/tdigest"
+bug-reports: "https://github.com/SGrondin/tdigest/issues"
+depends: [
+  "ocaml" { >= "4.10.0" }
+  "dune" { >= "1.9.0" }
+
+  "base" { >= "v0.17.0" & < "v0.18.0" }
+  "ppx_sexp_conv" { >= "v0.17.0" }
+
+  "ppx_expect" { >= "v0.17.0" & with-test }
+  "ppx_custom_printf" { >= "v0.17.0" & with-test }
+  # "ocamlformat" { = "0.25.1" } # Development
+  # "ocaml-lsp-server" # Development
```

Comment:
Why not use

```suggestion
  "ocamlformat" { = "0.25.1" & with-dev-setup }
  "ocaml-lsp-server" { with-dev-setup }
```

This allows you to install locally with `opam install --with-test --with-dev .`, but it won't get into the switch of users who install without `--with-dev`.

https://opam.ocaml.org/doc/Manual.html#pkgvar-with-dev-setup

It's not the most well-known of opam feature so maybe that's the reason why. But maybe there's another reason??

Reply:
This is great, I have wanted `--with-dev` for years! Glad to see it was added to opam.
Thank you